packages/store-ui/src/atoms/Icon/Icon.tsx

```diff
@@ -2,7 +2,13 @@ import type { HTMLAttributes, ReactNode } from 'react'
 import React, { forwardRef } from 'react'
 
 export interface IconProps extends HTMLAttributes<HTMLSpanElement> {
+  /**
+   * The icon component
```

Comment:
The pattern you want to ensure here is to have a description for each prop?

Reply:
How about this?

```suggestion
   * A React component that will be rendered as an icon.
```

Reply:
@lariciamota isn't a pattern is the way to document for storybook.

Reply:
@victorhmp thank you for the suggestion.